src/plays/netlify-card-game/NetlifyCardGame.jsx

```diff
@@ -0,0 +1,229 @@
+        const findPair = imageItems.find(
+          (i) => i.img === currentlyShownItem.img
+        );
+        if (!findPair) {
+          // if not same, then find the item from the main array
+          const findClickedItem = imageItems.find((i) => i.id === id);
+          findClickedItem.show = true; // need to display it for sometime
+          const updatedItemList = newImgArray.map((i) =>
+            i.id === findClickedItem.id ? findClickedItem : i
+          );
+          setNewImgArray(updatedItemList); // updating the main array coz we need to show a glimpse of the clicked item
+          disableClick.current = true; // while rage clicking state update taking time so we need to disable click
+          return timeOutCall(findClickedItem, currentlyShownItem); // timeout will hide the item after 1 second
+        } else {
+          // if user clicks the same imgage of the active image then we find which one is clicked specefically by id this time
+          disableClick.current = true; // disable the click
+          const otherPair = imageItems.find(
+            (i) => i.id !== currentlyShownItem.id
+          ); // searching for same image other object
+          otherPair.show = true; // have to render it
+          const updatedItemList = newImgArray.map((i) =>
+            i.id === otherPair.id ? otherPair : i
+          ); // updated array maintaining the index
+          duplicateImgArray.current = updatedItemList.filter(
+            (i) => i.show === false
+          ); // updating the duplicate array
+          setNewImgArray(updatedItemList); // updating the main object
+          setMatchedItems([...matchedItems, findPair]);
+          disableClick.current = false;
+        }
+      } else {
+        const findClickedItem = newImgArray.find((i) => i.id === id);
+        findClickedItem.show = true;
+        const updatedArray = newImgArray.map((i) =>
+          i.id === findClickedItem.id ? findClickedItem : i
+        );
+        setNewImgArray(updatedArray);
+      }
+    };
+
+  const timeOutCall = (clickedItem, activeItem) => {
+    return setTimeout(() => {
+      clickedItem.show = false;
+      if (activeItem) {
+        activeItem.show = false; // for the first time when user clicks the current object can be undefined so have to check it
+      }
+      const updatedArray = newImgArray.map((i) => {
+        if (i.id === clickedItem.id) {
+          return clickedItem;
+        } else if (activeItem && i.id === activeItem?.id) {
+          return activeItem;
+        } else {
+          return i;
+        }
+      });
+      disableClick.current = false;
+      setNewImgArray(updatedArray);
+    }, 2000);
+  };
+
+  const resetHandler = () => {
+    // resetting the entire game
+    setBoardStats({ ...initialState });
+    setMatchedItems([]);
+    shuffle();
+  };
+
+  return (
+    <>
+      <div className='play-details'>
+        <PlayHeader play={play} />
+        <div className='play-details-body'>
```

Comment:
Add a play specific class name here. Prefix that class name in all your style definition. It will make sure there is no conflict of the file name.

We are planning to do this change at the framework level to take care automatically. However, until that is in, please do this change.

Reply:
its done